tutorials/llama_index_knowledge_base_tutorial.ipynb

```diff
@@ -0,0 +1,402 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "_X9GuXoSXleA"
+   },
+   "source": [
+    "<center>\n",
+    "    <p style=\"text-align:center\">\n",
+    "        <img alt=\"phoenix logo\" src=\"https://storage.googleapis.com/arize-assets/phoenix/assets/phoenix-logo-light.svg\" width=\"200\"/>\n",
+    "        <br>\n",
+    "        <a href=\"https://docs.arize.com/phoenix/\">Docs</a>\n",
+    "        |\n",
+    "        <a href=\"https://github.com/Arize-ai/phoenix\">GitHub</a>\n",
+    "        |\n",
+    "        <a href=\"https://join.slack.com/t/arize-ai/shared_invite/zt-1px8dcmlf-fmThhDFD_V_48oU7ALan4Q\">Community</a>\n",
+    "    </p>\n",
+    "</center>\n",
+    "<h1 align=\"center\">Improving Your Knowledge Base</h1>\n",
+    "\n",
+    "Imagine you've built and deployed an LLM question-answering service that enables users to ask questions and receive answers from a knowledge base. You want to understand what kinds of questions your users are asking and whether you're providing good answers to those questions.\n",
+    "\n",
+    "Phoenix helps you pinpoint user queries that are not answered by your knowledge base so that you know which topics to iterate and improve upon. As you'll see, your users are asking questions on several topics that your knowledge base does not cover.\n",
+    "\n",
+    "In this tutorial, you will:\n",
+    "\n",
+    "- Download an pre-indexed knowledge base and run a LlamaIndex application\n",
+    "- Download user query data and knowledge base data, including embeddings computed using the OpenAI API\n",
+    "- Define a schema to describe the format of your data\n",
+    "- Launch Phoenix to visually explore your embeddings\n",
+    "- Investigate clusters of user queries with no corresponding knowledge base entry\n",
+    "\n",
+    "⚠️ This notebook requires an [OpenAI API key](https://platform.openai.com/account/api-keys).\n",
```

Comment:
this makes it seem like a show-stopper, but in fact it's mostly for explaining the background and context, so it's seems like a bit of a diversion from the main thrust of this notebook, which is showcasing Phoenix.